src/loader/cmi.ml

```diff
@@ -524,12 +541,18 @@ let read_label_declaration env parent ld =
   let type_ = read_type_expr env ld.ld_type in
     {id; doc; mutable_; type_}
 
-let read_constructor_declaration_arguments env parent arg =
+let read_constructor_declaration_arguments env _parent arg =
```

Comment:
Looks like `_parent` should be restored back to `parent`, since it's used in the 4.03 code. I recommend adding `ignore parent;` to the 4.02 code to suppress the warning.

Reply:
Fixed 🆗